src/ehr_components/history/index.tsx

```diff
@@ -0,0 +1,181 @@
+import { toastr } from "react-redux-toastr";
```

Comment:
I prefer to have history.tsx as index.tsx we can have a common data folder with section. we can discuss that

Reply:
Done. make sense. moved back as `history.tsx` and moved data to `data/history.js`